spec/models/question_spec.rb

```diff
@@ -40,4 +40,22 @@
   describe 'attributes' do
     it { expect(question.answers).to include(*question.correct_answers) }
   end
+
+  describe 'public instance methods' do
+    context 'responds to its methods' do
+      it { expect(question).to respond_to(:correct_count) }
+      it { expect(question).to respond_to(:correct_ratio) }
+      it { expect(question).to respond_to(:total_rates) }
+      it { expect(question).to respond_to(:like_ratio) }
+    end
+
+    context 'executes methods correctly' do
+      context 'self.correct_ratio' do
```

Comment:
1) Those methods belong to the model.
2) To set the inputs for the below method
`def correct_count
    GivenAnswer.where(question_id: id, is_correct: true).count
  end`
should we create instances of Given Answer for the test or stub/mock what it returns ?